playbooks/openshift-checks/private/health.yml

```diff
@@ -3,10 +3,28 @@
   hosts: oo_all_hosts
 
   roles:
+  - lib_openshift
   - openshift_health_checker
   vars:
   - r_openshift_health_checker_playbook_context: health
   post_tasks:
+  - name: Get cluster resources
+    delegate_to: "{{ groups.oo_first_master.0 }}"
+    run_once: True
+    oc_obj:
+      state: list
+      kind: "{{ item }}"
+      all_namespaces: True
+    register: resources
+    with_items:
+    - nodes
+    - pods
+    - services
+    - endpoints
+    - routes
+    - clusternetworks
+    - hostsubnets
+    - netnamespaces
```

Comment:
Can we use `import_playbook: health.yml' in adhoc.yml instead of duplicating the code?

Reply:
I think it would be the other way around: `health.yml` could set `openshift_checks` to `['@health']` and then import the `adhoc.yml` playbook.  However, the imported playbook would also override the importing playbook's `r_openshift_health_checker_playbook_context` setting.  It might be simpler to put the task in a new, separate file and include the new file in both `adhoc.yml` and `health.yml`.